pkgs/development/python-modules/distlib/default.nix

```diff
@@ -18,6 +18,10 @@ buildPythonPackage rec {
     setuptools
   ];
 
+  postInstall = ''
```

Comment:
Might be nice to make it conditional to non-windows platforms like in https://github.com/NixOS/nixpkgs/pull/223776/files